libs/langchain/langchain/llms/arcee.py

```diff
@@ -30,7 +30,7 @@ class Arcee(LLM):
     _client: Optional[ArceeWrapper] = None  #: :meta private:
     """Arcee _client."""
 
-    arcee_api_key: str = ""
+    arcee_api_key: Union[Optional[str], SecretStr]
```

Comment:
any reason not to make this just SecretStr

Reply:
I think the standard way of init the acree model with the constructor is with a str. Then it must contains a Secret string once casted with the validation function.